lib/src/components/NxSmallTag/NxSmallTag.scss

```diff
@@ -33,58 +35,67 @@
     padding: 0 var(--nx-spacing-2x);
     white-space: nowrap;
   }
+}
 
-  &--blue {
-    .nx-small-tag__text {
-      background-color: var(--nx-swatch-blue-30);
-    }
+.nx-small-tag--blue {
```

Comment:
Why'd you change the syntax here?

Reply:
> Why'd you change the syntax here?

reverted to previous syntax: [7b2ffb4](https://github.com/sonatype/sonatype-react-shared-components/pull/1152/commits/7b2ffb42b20dd55b7698f716bdb96bc1cfade1c9)